### DISCUSSION TITLE
Install old version of node

### DISCUSSION THREAD

**Discussion by datbilling** [hassio-addons/addon-node-red#1550] [Q&A] [answered]
Is it possible to install an older version of a node? 

I know there's an npm command but I don't think you can run commands if using the addon?

I saw you can specify packages in the addon configuration but the documentation doesn't mention if you can include a version there?

**Comment by ndoggac** [comment]
Would love to know this too.  Believe there's a feature request in HA-core for this.  I would like to roll back to 14.0.1 for NR to see if will eliminate this latest 502 bad gateway issue.  My backups only go back to 14.0.2.  I assume it was a change in HA-core that caused the issue, so maybe the fix needs to be there?

**Comment by frenck** [comment, marked as answer]
That is not possible with this addon.
